docker/FileCheckTests/compile_time_patterns

```diff
@@ -0,0 +1,4 @@
+; COM: Checks the pattern of compile time log file. Makes sure the header
+; COM: and the data rows' format are as expected.
```

Comment:
I think it will work even without `COM:`. There's nothing this could be confused it as this file is only passed to FileCheck.